components/card/index.jsx

```diff
@@ -118,6 +120,10 @@ Card.propTypes = {
 	 * Footer often contains pagination.
 	 */
 	footer: PropTypes.node,
+	/**
+	 * Allows card to have no header.
+	 */
+	hasNoHeader: PropTypes.bool,
```

Comment:
Should we try to keep prop names in the positive form, `hasHeader` and set it default to `true`? If not, maybe `forceNoHeader` might better explain this prop since it's really overriding `props.header` and forcing the component to not have a custom or default header.

Reply:
No truthy defaults: https://github.com/salesforce/design-system-react/blob/master/docs/codebase-overview.md#no-truthy-defaults-for-boolean-type-props

Maybe we could add to the comment that it ignores `props.header` and other props?

Reply:
ah ok - good stuff about no truthy defaults. Yes, adding a comment on the prop will help.  I wanted to avoid naming props in the negative form so we don't end up with double negation like `hasNoHeader=false` but I guess that's ok. 

It would have been nice to just have `prop.header` to control the header but things get more complicated when we want to have the option of a default header. Cool! Let's leave as is and add a comment.

Reply:
I have added suggested prop description to the comment. @interactivellama please check.